Bucketeer2/Sources/Internal/Remote/Response/GetEvaluationsResult.swift

```diff
@@ -0,0 +1,6 @@
+import Foundation
+
+enum GetEvaluationsResult {
+    case success(GetEvaluationsResponse)
+    case failure(error: BKTError, featureTag: String)
+}
```

Comment:
The same structure as https://github.com/ca-dp/bucketeer-ios-sdk/pull/84#discussion_r979242287